BE/src/main/java/com/example/assig1/command/CommandHandler.java

```diff
@@ -0,0 +1,74 @@
+package com.example.assig1.command;
+
+
+import com.example.assig1.dto.QuestionDTO;
+import com.example.assig1.exception.UserNotFoundException;
+import com.example.assig1.model.Question;
+import com.example.assig1.model.User;
+import com.example.assig1.service.QuestionService;
+import com.example.assig1.service.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+@RequiredArgsConstructor
+@Component
+public class CommandHandler {
```

Comment:
This is not how the command pattern looks like! See https://www.baeldung.com/java-command-pattern